### DISCUSSION TITLE
코너 자리에 가서 앉았는데 나올 수가 없어요

### DISCUSSION THREAD

**Discussion by mj-ch** [metamong-friends/metamong-friends.github.io#59] [제보하기]
멋진 코너 자리에 앉았다가 회의실 이동하려고 나오려는데 나올 수가 없습니다. 
x누르면 그대로 자리에 앉아버려요

https://github.com/user-attachments/assets/e37a18bc-4f44-45e8-8e69-91c36f7a6cff

**Comment by prism27** [comment]
안녕하세요!!

아이고.. 가구 배치에 좀 더 신경을 썼어야 되는데, 끼어서 불편하셨겠네요.
office 전체적으로 확인해 조치할 수 있도록 하겠습니다.

동영상 까지 녹화해서 상세한 내용 제보해주셔서 정확한 위치, 상황 확인에  큰 도움이 되었습니다. 
감사합니다!!

**Comment by shlee303** [comment]
오늘 배포로 해당 현상 조치 했습니다! 제보 해주셔서 감사합니다 :) @mj-ch
